cmd/node/member/node/member-node.go

```diff
@@ -337,6 +339,28 @@ func (m *MemberNode) setupHandlers(
 	mediaRepo := database.NewMediaRepo(db)
 	notificationRepo := database.NewNotificationsRepo(db)
 
+	// Initialize CRDT statistics store
+	var crdtStore *crdt.CRDTStatsStore
+	// Type assert to get access to underlying Gossip
+	type gossipProvider interface {
+		Gossip() *pubsub.Gossip
+	}
+	if gp, ok := m.pubsubService.(gossipProvider); ok {
```

Comment:
no, that bad design
there's no reason to do type casting
you need to rethink design and interfaces